tests/kernel/timer/timer_api/src/main.c

```diff
@@ -248,6 +248,7 @@ void test_timer_periodicity(void)
 	 * whatever duration that was set.
 	 */
 	k_timer_status_sync(&periodicity_timer);
+	k_timer_status_sync(&periodicity_timer);
```

Comment:
I suggest leaving a comment explaining why this was called twice

Reply:
Done.